### PR TITLE
feature(main): refactor: split ssh cmd into exec and scp(#996)

### PR DESCRIPTION
**Pull request by cuisongliu**
[SKIP CI]seaols: 一句话简短描述该PR内容

### CODE REVIEW

**Review by SignorMercurio — APPROVED** [review]
Shall we delete `pkg/ssh/ssh.go`?

**Comment by cuisongliu** [comment]
> Shall we delete `pkg/ssh/ssh.go`?

ok

File: pkg/utils/ssh/exec.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+			}
+			err := sshClient.CmdAsync(ip, cmd)
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+	}
+	if err := eg.Wait(); err != nil {
+		return fmt.Errorf("failed to sealos exec command, err: %v", err)
```
Comment (fanux) [inline pkg/utils/ssh/exec.go:71]: failed to exec command

File: pkg/utils/ssh/exec.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			}
+			err := sshClient.Copy(ip, srcFilePath, dstFilePath)
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+	}
+	if err := eg.Wait(); err != nil {
+		return fmt.Errorf("failed to sealos copy command, err: %v", err)
```
Comment (fanux) [inline pkg/utils/ssh/exec.go:93]: failed to exec command

Comment (cuisongliu) [reply pkg/utils/ssh/exec.go:93]: this is scp logic , is not exec logic .

**Review by fanux — APPROVED** [review]

**Pull request merged**